sljit_src/sljitExecAllocator.c

```diff
@@ -94,7 +94,6 @@ static SLJIT_INLINE void free_chunk(void *chunk, sljit_uw size)
 #else /* POSIX */
 
 #ifdef __APPLE__
-#include <AvailabilityMacros.h>
```

Comment:
since there is only one macOS version that runs in arm, there is no need to look at the version, obsoleting this